packages/theme/src/ShadowRoot/Portal.tsx

```diff
@@ -47,7 +47,8 @@ export function usePortalShadowRoot<T>(renderer: (container: HTMLElement | undef
         }
 
         const container = shadow.appendChild(document.createElement('main'))
-        sheet.addContainer(shadow)
+
+        sheet?.addContainer(shadow)
```

Comment:
Please align the typing with `sheet`. You might forget to use the provider of context to wrap the component tree.